distribution/src/config/opensearch.yml

```diff
@@ -92,7 +92,7 @@ ${path.logs}
 # cluster.remote_store.enabled: true
 #
 # Repository to use for segment upload while enforcing remote store for an index
-# cluster.remote_store.repository: my-repo-1
+# cluster.remote_store.segment.repository: my-repo-1
 #
 # Controls whether cluster imposes index creation only with translog remote store enabled
 # cluster.remote_store.translog.enabled: true
```

Comment:
nit: this is not required anymore, right?

Reply:
yeah, let me remove